_posts/2019-06-25-lambda-quickstart.md

```diff
@@ -0,0 +1,320 @@
+   --from-literal=AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID \
+   --from-literal=AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY
+```
+
+#### 2.5. Deploy OpenFaaS helm chart
+
+
+##### 2.5.1 Gather Helm configuration arguments
+
+| Parameter                        | Description |
+| -------------------------------- | ----------- |
+| aws_region                       | AWS region that your Lambda functions will run in. For example `eu-west-1` |
+| faaslambda.lambda_execution_role | IAM role to assign each function. See step 2.3. for more information   . |
+| access_token                     | Early access token obtained from step 1. |
+| access_email                     | Email address used when registering token from step 1. |
+
+
+##### 2.5.2 Deploy chart
+
+Below are the commands to clone and deploy the official OpenFaaS Helm chart, configured to deploy to AWS Lambda.
+Please change the `--set` parameters `faaslambda.aws_region`, `faaslambda.lambda_execution_role`, `faaslambda.access_token` and
+`faaslambda.access_email` with the values you gathered together in the previous step.
+
+```bash
+$ git clone git@github.com:openfaas/faas-netes.git
+cd faas-netes
+helm repo add openfaas https://openfaas.github.io/faas-netes/
+helm repo update \
+ && helm upgrade openfaas --install openfaas/openfaas \
+                                     --namespace openfaas  \
+                                     --set faaslambda.create=true \
+                                     --set functionNamespace=openfaas-fn \
+                                     --set faaslambda.aws_region=eu-west-1 \
+                                     --set faaslambda.lambda_execution_role=arn:aws:iam::123456789012:role/openfaas-function-role \
+                                     --set faaslambda.access_token=eyJhbGciOiJFUzI1NiIsImtpZCI6IktrVnV5a2VDMkJCenRUREVsSzh5XzNzNlludlBiTHAtRmQ1djY2TGxvUU0iLCJ0eXAiOiJKV1QifQ.eyJhdWQiOiJjb29sZ3V5QGdtYWlsLmNvbSIsImV4cCI6MTU2ODU0NDQyMywiaWF0IjoxNTYwNTk1ODMwLCJpc3MiOiJmYWFzLWxhbWJkYSIsIm5iZiI6MTU2MDU5NTgzMCwic3ViIjoiZWFybHktYWNjZXNzIn0.0cKLO7nkjGQYwjNaQY2OLJFLYpz0bHxB-9btAU9TRrblOO8-hpz1pDRSajwg_SiY0IVRVZfaJRLFqeVy4FS8dA \
+                                     --set faaslambda.access_email=fredjones@gmail.com
+```
+
+### 3. Verifying the installation
+
+#### 3.1 Main OpenFaaS components are running
+
+```bash
+$ kubectl get pods -n openfaas
+
+kubectl get pods -n openfaas                                                                 290ms  Wed 26 Jun 08:06:38 2019
+NAME                            READY   STATUS    RESTARTS   AGE
+alertmanager-775d5cd49f-2sf2b   1/1     Running   0          8d
+faas-idler-57b849dfd-k9rfv      1/1     Running   0          8d
+gateway-59d9dbbf-6znm2          2/2     Running   0          8d
+nats-75d8f56846-j2b4b           1/1     Running   0          8d
+prometheus-5945df7857-rt2fw     1/1     Running   0          8d
+queue-worker-5f96b567c5-pnsgd   1/1     Running   0          8d
+
+$ faas-cli version
```

Comment:
Explain about the provider name that people see on "faas-cli version" ? New heading "Check the version / provider"?